Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/Survey.kt

```diff
@@ -34,7 +34,9 @@ data class Question(
     val id: Int,
     @StringRes val questionText: Int,
     val answer: PossibleAnswer,
-    @StringRes val description: Int? = null
+    @StringRes val description: Int? = null,
+    val permissionsRequired: List<String>? = null,
```

Comment:
How about initialising this with `emptyList()` and making it non-null?
Nullability doesn't bring anything to empty list.

Reply:
Done!